content/news/051/index.md

```diff
@@ -69,6 +69,24 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Way of Rhea][wor]
+
+[![Way of Rhea](wor.jpg)][wor]
```

Comment:
The image is missing a "flavor text" (_in italic_)

I think it's important because I assume those help with accessibility, even though an alt would be better, but proper "alt" is outside this PR scope.

Reply:
Done